libevm/VMCalls.cpp

```diff
@@ -130,7 +130,11 @@ void VM::caseCreate()
 		if (!m_schedule->staticCallDepthLimit())
 			createGas -= createGas / 64;
 		u256 gas = createGas;
-		m_SPP[0] = (u160)m_ext->create(endowment, gas, bytesConstRef(m_mem.data() + initOff, initSize), m_onOp);
+		h160 addr;
+		owning_bytes_ref revertOutput;
+		std::tie(addr, revertOutput) = m_ext->create(endowment, gas, bytesConstRef(m_mem.data() + initOff, initSize), m_onOp);
+		m_SPP[0] = (u160)addr;
+		m_returnData = revertOutput.toBytes();
```

Comment:
No. Executive handles this. For CREATE, the VM output is only taken in case of REVERT. https://github.com/ethereum/cpp-ethereum/blob/develop/libethereum/Executive.cpp#L411.